### PR TITLE
English grammar

### PR DESCRIPTION
**Pull request by pzavoral**

### CODE REVIEW

File: 05-RemoveFile/README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@
 * Next you have to commit your changes.
 * Directories are not tracked by _GIT_.
   * Once a directory becomes empty it will not even appear in your sandbox.
-  * This can be a difference (for example compared to TFS). If you would keep
-    a folder for some reason create a file inside. Commonly used files are
+  * This can be a difference (for example compared to TFS). If you want to keep
+    a folder for some reason, create a file inside. Commonly used files are
     ```.gitattribute``` (this has special meaning), ```.gitkeep```,
-    ```.placeholder``` or ```.gitignore``` (this has special meaning).
+    ```.placeholder```, or ```.gitignore``` (this has special meaning).
```
Comment (graywolf) [inline 05-RemoveFile/README.md:13]: should there really be a comma?

**Comment by graywolf** [comment]
Overall good, after Kyle gives this his blessing, I'll merge it all together.

File: 00-Overview/readme.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+* there usually is one `upstream`, but which repository it is depends on agreement between project members
 * this has some (maybe) unexpected side effects
 	* you cannot restrict read access to specific files, folders, branches; if you want someone to clone from you, he has to be able to get everything
 	* everyone has complete history of the project
 
 All repositories are equal
 --------------------------
 
-* while you do clone from somewhere, you are not restricted to pushing/pulling from there
+* while you do clone from somewhere, you are not restricted to pushing to/pulling from there
```
Comment (KyleFord76063) [inline 00-Overview/readme.md:18]: Not sure the "to" is needed.  Could rephrase like this:
"...not restricted to "pushing / pulling" from there"

File: 00-Overview/readme.md [hunk 3]
```diff
@@ -32,26 +32,26 @@ Everyone has complete history
 
 * this is very important point
 * I'm sure it happened to you at least once, you commit password into source control
-	* if someone already cloned/pulled from you, all you can do is change it
+	* if someone already cloned/pulled from you, all you can do is to change it
```
Comment (KyleFord76063) [inline 00-Overview/readme.md:35]: I would not add "to" here

File: 00-Overview/readme.md (hunk 2 above)
Comment (graywolf) [inline 00-Overview/readme.md:18]: On , KyleFord76063 wrote:

> [..]
> 
> Not sure the "to" is needed.  Could rephrase like this:
> "...not restricted to "pushing / pulling" from there"

I like the Kyle's version more :)

## 

There are only two hard things in Computer Science:
cache invalidation, naming things and off-by-one errors.

File: 00-Overview/readme.md [hunk 4]
```diff
@@ -32,26 +32,26 @@ Everyone has complete history
...
 	* in other words, **don't ever rewrite history unless you know what you are doing**
 
 Commit & history integrity
 --------------------------
 
-* in git you don't have revision numbers (as in TFS or SVN), instead git uses SHA-1 hashes to mark commits
-* each file in your repository is represented by it's SHA-1 checksum
-* part of chechsummed git commit is checksum of previous commits, so it forms cryptographically secure line from root commit to current state
-	* this means that if you have repository at the same commit as someone else and no local changes, you can be (pretty) sure you have the exact same files
+* unlike to TFS or SVN, there are no revision numbers in git, git uses SHA-1 hashes to mark commits instead
```
Comment (KyleFord76063) [inline 00-Overview/readme.md:46]: "unlike TFS or SVN," -- remove the "to" here as well

File: 05-RemoveFile/README.md (hunk 1 above)
Comment (pzavoral) [inline 05-RemoveFile/README.md:13]: See http://www.grammarbook.com/punctuation/commas.asp Rule 1

File: 04-FirstLab/README.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
-* Watch the _PowerShell_ command line how it changes.
+* Watch how the command line is changing.
 * Use ```git status``` to see what is going on.
 
 # Task
 
-* Create in your repository following folders and files.
-* To files put some content per your preference:
+* Create the following folders and files in your repository.
+* Write some content per your preference into the files:
```
Comment (KyleFord76063) [inline 04-FirstLab/README.md:10]: I also rephrased this in PR #28.
What you have is also better than what was there before.  Either one.

File: 04-FirstLab/README.md [hunk 6]
```diff
@@ -49,5 +49,5 @@ Date:   Thu Mar 17 12:56:25 2016 +0100
 
 # Comments
 
-* When adding content of folder you do not have to add individual files.
+* When adding content of a folder, you do not have to add individual files.
```
Comment (KyleFord76063) [inline 04-FirstLab/README.md:52]: I meant to comment that sometimes "folder" is used, and sometimes "directory" is used.  Might try to choose one and stick with it.

File: 05-RemoveFile/README.md [hunk 7]
```diff
@@ -7,14 +7,14 @@
 * Next you have to commit your changes.
 * Directories are not tracked by _GIT_.
   * Once a directory becomes empty it will not even appear in your sandbox.
-  * This can be a difference (for example compared to TFS). If you would keep
-    a folder for some reason create a file inside. Commonly used files are
+  * This can be a difference (for example compared to TFS). If you want to keep
```
Comment (KyleFord76063) [inline 05-RemoveFile/README.md:10]: I rephrased this in PR #28 also.

File: 06-Branch/readme.md [hunk 8]
```diff
@@ -6,15 +6,15 @@ git-branch
 We are continuing on repository from part 04, so let's list branches
 we have:
 
-	$ git branch [--list]
+	$ git branch
```
Comment (KyleFord76063) [inline 06-Branch/readme.md:9]: I like this.  Remove it unless you talk about "--list".

File: 06-Branch/readme.md [hunk 9]
```diff
@@ -29,7 +29,7 @@ we just renamed the branch to fix it
 	* master
 	  my_apple_app
 
-And you know what? Let's not do OS X version of our application, since we don't know how
+And you know what? Let's not to do OS X version of our application, since we don't know how
```
Comment (KyleFord76063) [inline 06-Branch/readme.md:32]: Remove "to". This edit does not help.

File: 06-Branch/readme.md [hunk 10]
```diff
@@ -39,14 +39,14 @@ So, to sum up what we've just learned
...
+* git branch _BRANCH_NAME_
+	* create branch named _BRANCH_NAME_ from current HEAD
+* git branch -m _OLD_BRANCH_NAME_ _NEW_BRANCH_NAME_
 	* rename branch
-* git branch -d $BRANCH_NAME
+* git branch -d _BRANCH_NAME_
 	* delete branch
 
-More informations
+More information
```
Comment (KyleFord76063) [inline 06-Branch/readme.md:49]: Remember to remove one of the dashes below this too!  (I also fixed this in PR #28)

**Comment by pzavoral** [comment]
I will rebase it and post it as a new pull request

**Pull request closed without merging**